javascript/javascript-5.md

```diff
@@ -55,3 +56,10 @@ Refer back to the movie night activity from the JavaScript 3 lesson. Refactor yo
 Complete these HackerRank Challenges:
 - [HackerRank challenge: Switch](https://www.hackerrank.com/contests/7days-javascript/challenges/js-switch-statements) which deals with a simple switch statement.
 - [Conditional Statements: Switch](https://www.hackerrank.com/challenges/js10-switch)
+
+### Check for Understanding
+
+- Write down the syntax of switch statement in JavaScript.
+- What will happen if break is not used in switch case?
+- What is the advantage of using switch case over if/else?
+- Using switch case write a program to accept a letter in lower case and create an alert 'Letter is a vowel' if the letter is a vowel.
```

Comment:
Using a switch statement, write a program to accept a lower case letter.  Create an alert with the text 'This letter is a vowel' if the letter is a vowel.